tests/unit/classifier/LDA_unittest.cc

```diff
@@ -201,10 +202,11 @@ TEST(LDA, num_classes_in_labels_exception)
 	auto labels = some<CDenseLabels>();
 	labels->set_labels(lab);
 	auto features = some<CDenseFeatures<float64_t>>(feat);
-	auto lda = some<CLDA>(0, features, labels, SVD_LDA);
+	auto lda = some<CLDA>(0, SVD_LDA);
+	lda->set_labels(labels);
 	// should throw an incorrect number of classes exception (expected value is
 	// 2)
-	EXPECT_THROW(lda->train(), ShogunException);
+	EXPECT_THROW(lda->train(features), ShogunException);
```

Comment:
we need some tests for the dispatcher in a mock class environment, to test that

* features are correctly dispatched, also for multiple feature types
* errors are thrown if the "wrong" type is passed